lib/active_model/serializer.rb

```diff
@@ -177,6 +177,10 @@ def json_key
       end
     end
 
+    def as_json
+      self.class.adapter.new(self).as_json
```

Comment:
Sounds good to me... as a regular user of AMS, all I care is that I -can- call `#as_json` or `#to_json`, not what happens behind the curtains to enable it :grinning: